diff --git a/node_modules/node-gyp-build/index.js b/node_modules/node-gyp-build/index.js
index b5096ed..6b0a3f9 100644
--- a/node_modules/node-gyp-build/index.js
+++ b/node_modules/node-gyp-build/index.js
@@ -61,10 +61,18 @@ load.path = function (dir) {
   function resolve (dir) {
     // Find most specific flavor first
     var prebuilds = path.join(dir, 'prebuilds', platform + '-' + arch)
+    var prebuildsZMQ = path.join(dir, 'node_modules', 'zeromq', 'prebuilds', platform + '-' + arch);
+
     var parsed = readdirSync(prebuilds).map(parseTags)
     var candidates = parsed.filter(matchTags(runtime, abi))
     var winner = candidates.sort(compareTags(runtime))[0]
     if (winner) return path.join(prebuilds, winner.file)
+
+
+    var parsedZMQ = readdirSync(prebuildsZMQ).map(parseTags)
+    var candidatesZMQ = parsedZMQ.filter(matchTags(runtime, abi))
+    var winnerZMQ = candidatesZMQ.sort(compareTags(runtime))[0]
+    if (winnerZMQ) return path.join(prebuildsZMQ, winnerZMQ.file)
   }
 }
 
